### PR TITLE
Moddataimp 483 increase test coverage for mod data import

### PR DESCRIPTION
**Pull request by bisecomsergiy**
As per the current task, the module is covered by tests, removed initial code smells.

### CODE REVIEW

File: src/test/java/org/folio/service/processing/kafka/WriteStreamWrapperTest.java [hunk 1]
```diff
@@ -0,0 +1,131 @@
+package org.folio.service.processing.kafka;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Future;
+import io.vertx.core.Handler;
+import io.vertx.kafka.client.producer.KafkaProducer;
+import io.vertx.kafka.client.producer.KafkaProducerRecord;
+//import org.junit.jupiter.api.Assertions;
```
Comment (viktor-draban) [inline src/test/java/org/folio/service/processing/kafka/WriteStreamWrapperTest.java:8]: You should not include commented code in PR

Comment (viktor-draban) [reply src/test/java/org/folio/service/processing/kafka/WriteStreamWrapperTest.java:8]: It's about the whole class

Comment (VRohach) [reply src/test/java/org/folio/service/processing/kafka/WriteStreamWrapperTest.java:8]: @viktor-draban Thx! But we'll remove this class at all

Comment (bisecomsergiy) [reply src/test/java/org/folio/service/processing/kafka/WriteStreamWrapperTest.java]: Thank you, it is fixed

**Review by VRohach — APPROVED** [review]

**Review by KaterynaSenchenko — APPROVED** [review]

**Review by Aliaksandr-Fedasiuk — APPROVED** [review]

**Pull request merged**